src/renderer/l10n/cn.js

```diff
@@ -1,278 +1,265 @@
+  priceUc: '价格',
+  print: '打印',
+  privateKey: '私人密钥',
+  properlyBackedUpWallet: '如果您没有正确备份此钱包的密钥。',
+  publicAddress: '公共地址',
+  purchaseNow: '现在购买',
+  receivedNEO: '收到NEO的转账确认。',
+  recentTransactions: '最近的交易',
+  receive: '接收',
+  recipient: '接收者',
+  regressionTrend: '回归趋势',
+  relayedToNetwork: '{amount} {symbol} {action} 传送到网络。',
+  remove: '移除',
+  repeatYourPassphrase: '请在此处再输入一遍密码',
+  save: '保存',
+  saveAndBackupTheKeys: '请保存并备份以下密码。',
+  savedWallet: '保存钱包',
+  saving: '保存中......',
+  search: '搜索',
+  secondsAgo: '{seconds} 几秒钟前',
+  selectAWallet: '选择钱包',
+  selectIco: '选择ICO',
+  sell: '出售',
+  send: '发送',
+  sendGasClaim: '发出 GAS 的索取声明。',
+  sendingClaim: '正在发出 GAS 索取声明 ~为 {amount}GAS。',
+  sentClaim: '已发出GAS索取声明 ~为 {amount}GAS。',
+  setMarketRelayed: 'setMarket（设置市场） 调用转发',
+  settings: '设置',
+  size: '规格',
+  spread: '传播',
+  status: '状态',
+  steps: '步骤：',
+  submit: '提交',
+  submittingMultipleTimesWarning: '多次提交可能导致资金损失；',
+  success: '成功！',
+  systemFee: '系统费用',
+  testDEXVersion: '您正在观看Aphelion DEX测试版本。网站主页即将推出。',
+  theDexContractHasBeenUpdated: `DEX合约已更新，相应的钱包升级才能继续使用DEX。
+  请使用以下控件取消订单，合同中的资金将会退回到您的钱包里，然后下载最新版本。`,
+  theFeeForCompletingYourTradeWillBe: '完成交易的费用将是 {amount} APH',
+  theFeeForCompletingYourTradeWillDepend: `完成交易的费用将取决于匹配的最终报价数量，
+  但将介于 {min} APH和{max} APH之间`,
+  thisIsAPreliminaryDemo: `这是Aphelion DEX的初步演示和测试版本。它在 Neo 测试网上运行，
+  但可能会表现出已降低的性能，网站主页即将推出。`,
+  thisMeansThatItIsIneligible: '这意味着它不符合只需公布的订单的要求。',
+  thisOrderRequires: `此订单需要{quantity} {symbol} 来完成。
+  您当前的合约余额是{balance} {symbol}。提交此订单需首先提交 {deposit}
+   {symbol}以便能成功处理。`,
+  thisOrderWouldTakeTheFollowing: '此订单将接受以下报价：',
+  thisWillBeAMakerOrder: `这将会是一个已登记的制造商订单，在它被接受之前我们会一直保留。
+  此次交易我们不会收取您任何费用。`,
+  thisWillBeAMakerOrderLeftOnTheBook: `这将会是一个已登记的制造商订单，
+  在它被接受之前我们会一直保留。此次交易我们不会收取您任何费用。`,
+  time: '时间',
+  to: '到',
+  toDownload: '去下载',
+  token: '代币',
+  tokenSaleSuccessful: '代币销售参与成功。您的余额 {symbol} 现在是 {balance}',
+  tokenStats: '代币统计',
+  totalSupply: '总供应数',
+  tradeDEX: '交易 DEX',
+  tradeHistory: '交易历史',
+  tradesWillBePoweredBy: `交易将通过 Aphelion 的去中心化的交易资产分类 (D.E.A.L)
+   的智能合同，将 APH 的所有代币重新分配给所有 APH 的代币持有者。`,
+  transaction: '交易',
+  transferAllNEO: '将所有NEO转移到您自己的账户。',
+  transferredNEO: '已转移 {amount} NEO到您的账户。',
+  transferringNEO: '正在转移 {amount} NEO到您的账户。',
+  twentyFourHour: '24小时',
+  twentyFourHourChange: '24小时变化',
+  type: '种类，类型',
+  unableToReachNetwork: '无法访问网络',
+  unableToReachTradingServer: ' 无法连接交易服务器。',
+  unconfirmed: '未经证实',
+  unitsTotal: '单位总数',
+  useAnIcoScriptHash: '使用ICO脚本散列',
+  value: '价值',
+  vol: '成交量',
+  volume: '成交量',
+  waitForGasClaim: '等待GAS的索取声明确认。',
+  waitForGasClaimTransaction: '等待GAS的索取声明的交易细节。',
+  waitForNEO: '等待NEO转账确认。',
+  waitingForConfirmation: '正在等待确认......',
+  waitingForGasClaim: '正在等待GAS的索取声明确认。',
+  waitingForNEO: '正在等待NEO转账确认。',
+  waitingForTransaction: '正在等待交易在区块浏览器中出现......',
+  walletBalance: '钱包余额',
+  walletBalanceContractBalance: '钱包余额：{walletBalance}\n 合同余额: {contractBalance}',
```

Comment:
Same as my comment bellow; this needs to have a  parameter for the balance in open orders; see latest english file.